src/lib/accessibility/FocusZone/FocusZone.types.ts

```diff
@@ -53,6 +53,11 @@ export interface FocusZoneProps extends React.HTMLAttributes<HTMLElement | Focus
    */
   shouldFocusOnMount?: boolean
 
+  /**
+   * If a first focusable element should be force focused when FocusZone container receives focus.
+   */
+  shouldFocusFirstElementWhenReceivedFocus?: boolean
```

Comment:
What's default for this?  Seems like most times this would be desirable?

Reply:
The default is false. It is needed in many times, especially when we use nested focus zones, but I wouldn't say the most times. The most times actually we don't need it, for e.g. when we have a ```Chat```, which is a ```FocusZone``` itself, and ```chatMesssages```, which are the nested FocusZones. But when we navigate with Up/Down arrow keys, we still expect the focus goes to the message (FocusZone container) unless we press, for e.g., Enter key to go inside message and focus first focusable element there.

Reply:
I think the case of having focus zone in focus zone is rare enough that we can say consumers need to think about it. I'm fine with having it set to false.